go.mod

```diff
@@ -1,16 +1,28 @@
 module github.com/cloudflare/pat-go
 
-go 1.19
+go 1.23
+
+toolchain go1.23.6
 
 require (
 	github.com/cisco/go-hpke v0.0.0-20210524174249-dd22b38cf960
 	github.com/cloudflare/circl v1.3.7
+	github.com/quic-go/quic-go v0.50.0 // required for RFC 9000 varint
```

Comment:
I consider you can borrow code from stdlib instead of adding a large dependency just for the encoding.
https://cs.opensource.google/go/x/net/+/refs/tags/v0.36.0:internal/quic/quicwire/wire.go